testproject/Assets/Prefabs/PlayerCube.prefab

```diff
@@ -21,7 +21,7 @@ GameObject:
   - component: {fileID: -7468455824255952951}
   - component: {fileID: -4978466230159947418}
   m_Layer: 0
-  m_Name: Cube
```

Comment:
This name change is from Unity Editor. As far as I could see, the cube was already called `PlayerCube`.